File: 392. Is Subsequence/Is_Subsequence.cpp
class Solution {
public:
    bool isSubsequence(string s, string t)
    {
        int i, j, k = 0, flag = 0;
        int len_s = s.length();
        int len_t = t.length();
        for(i = 0; i < len_s; i++)
        {
            for(j = k; j < len_t; j++)
            {
                if(s[i] == t[j])
                {
                    flag++;
                    k = j + 1;
                    break;
                }
            }
        }
        return flag == len_s;
    }
};

File: 392. Is Subsequence/Is_Subsequence.py
class Solution:
    def isSubsequence(self, s: str, t: str) -> bool:
        k = 0
        flag = 0
        len_s = len(s)
        len_t = len(t)
        for i in range(len_s):
            for j in range(k, len_t):
                if s[i] == t[j]:
                    flag += 1
                    k = j + 1
                    break
        
        return flag == len_s
